src/components/organisms/ChatSidebar/components/VenueChat/VenueChat.tsx

```diff
@@ -21,7 +17,7 @@ export const VenueChat: React.FC<VenueChatProps> = ({ onAvatarClick }) => {
         messages={messagesToDisplay}
         sendMessage={sendMessage}
         deleteMessage={deleteMessage}
-        onAvatarClick={onAvatarClick}
+        onAvatarClick={openUserProfileModal}
```

Comment:
[NB] We could potentially also remove `onAvatarclick` from `ChatBox` and just have it use `useProfileModalControls` directly? Haven't looked where else this component is used and if this prop is used in a different way in those locations.